src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -194,7 +194,7 @@ private function parseDefinitions($content, $file)
                 }
             }
         } else {
-            $defaults = array();
+            $defaults = array('public' => true);
```

Comment:
Previous fix was not enough in case `_defaults: { public: true }` was set anyway... So updated.